docs/deployment/kubernetes.mdx

```diff
@@ -1,5 +1,198 @@
 ---
 title: "Kubernetes"
-description: "Deploy BeeAI on Kubernetes"
+description: "Deploy BeeAI to Kubernetes using Helm"
 ---
 
+## Prerequisites
+
+- Kubernetes cluster (local or cloud)
+- Helm 3.x installed
+- kubectl configured to access your cluster
+- BeeAI installed for post-deployment configuration
+
+## Installation
+
+### Step 1: Create Configuration File
+
+Create a value file with the minimum configuration:
+
+`config.yaml`:
+```yaml
+# If you want to include agents from the default catalog (change release/tag accordingly):
+externalRegistries:
+  public_github: "https://github.com/i-am-bee/beeai-platform@release-v0.2.0#path=agent-registry.yaml"
+# Your custom agents as docker images
+providers:
+  # e.g.
+  # - location: ghcr.io/i-am-bee/beeai-platform-agent-starter/my-agent:latest
+  - location: <docker-image-id>
+# Generate the encryption key:
+#  - using UV (https://docs.astral.sh/uv/getting-started/installation/)
+#   $ uv run --with cryptography python3 -c 'from cryptography.fernet import Fernet; print(Fernet.generate_key().decode())'
+#  - using python3 directly
+#   $ python3 -m pip install cryptography # (or use your preferred way to install the cryptography package)
+#   $ python3 -c 'from cryptography.fernet import Fernet; print(Fernet.generate_key().decode())'
+encryptionKey: "encryption-key-from-command"
+features:
+  uiNavigation: true
+# this requires passing an admin password to certain endpoints, you can disable auth for insecure deployments
+auth:
+  enabled: true
+  admin_password: "my-secret-password"
+# Help us improve the platform by sharing anonymized telemetry data
+telemetry:
+  sharing: true
+```
+
+### Step 2: Install the Chart
+
+Then install the chart using:
+
+```shell
+helm install -f config.yaml beeai oci://ghcr.io/i-am-bee/beeai-platform/beeai-platform-chart/beeai-platform:0.2.6
+```
+
+### Step 3: Setup LLM
+
+After the beeai-platform becomes ready, it's necessary to configure the LLM provider. We will use the `admin-password` you created earlier and your preferred LLM credentials, for example:
+
+```shell
+kubectl run curlpod --image=curlimages/curl -it --rm --restart=Never -- curl -X PUT \
+  beeai-platform-svc:8333/api/v1/variables \
+  -u beeai-admin:my-secret-password \
+  -H "Content-Type: application/json" \
+  -d '{
+    "env": {
+        "LLM_API_BASE": "https://api.openai.com/v1",
+        "LLM_API_KEY": "sk-...",
+        "LLM_MODEL": "gpt-4o"
+    }
+  }'
+```
+
+## Use the Platform
+
+Test that the platform is working:
+
+```shell
+# port-forward in a separate terminal
+kubectl port-forward svc/beeai-platform-svc 8333:8333 &
+```
+
+```shell
+beeai list
+beeai run chat hi
+```
+
+## Configuration Options
+
+### Security Settings
+
+For production deployments, ensure proper security:
```

Comment:
nitpick: I'm not that confident in the security yet without proper user and auth methods, so not sure if I'd use this phrasing